doc/guide/keys.md

```diff
@@ -1 +1,184 @@
-# Secret Management
+---
+comment: true
+title: Secret Management
+description: Keys, files, and other out-of-store paths with Wire Tool.
+---
+
+# {{ $frontmatter.title }}
+
+{{ $frontmatter.description }}
+
+::: warning
+
+Pushing keys to your local machine is currently unimplemented and is planned for
+v2.0.0.
```

Comment:
shouldn't this be a v1.0 thing? this is rather important, besides this question, this is a nicely written document.